src/GafferImage/ImageReader.cpp

```diff
@@ -305,25 +306,27 @@ void ImageReader::compute( ValuePlug *output, const Context *context ) const
 	if( output == intermediateColorSpacePlug() )
 	{
 		std::string userColorSpace = colorSpacePlug()->getValue();
-		std::string intermediateSpace = "oiioDefault";
+		std::string metadataSpace = "";
```

Comment:
I might not have explained very well, but what I was trying to say was that if a userColorSpace has been specified, we don't even need to get the metadata and do the search through it - we can skip all that work entirely.